opencti-platform/opencti-graphql/src/modules/workspace/workspace-domain.ts

```diff
@@ -74,39 +62,50 @@ export const getOwnerId = (workspace: BasicStoreEntityWorkspace) => {
   return (Array.isArray(workspace.creator_id) && workspace.creator_id.length > 0) ? workspace.creator_id.at(0) : workspace.creator_id;
 };
 
-export const objects = async (context: AuthContext, user: AuthUser, workspaceId: string, args: WorkspaceObjectsArgs) => {
-  const key = buildRefRelationKey(RELATION_HAS_REFERENCE);
-  const types = args.types ?? ['Stix-Meta-Object', 'Stix-Core-Object', 'stix-relationship'];
-  const filters = [{ key, values: [workspaceId] }, ...(args.filters || [])];
+export const objects = async (context: AuthContext, user: AuthUser, { investigated_entities_ids }: BasicStoreEntityWorkspace, args: WorkspaceObjectsArgs) => {
+  const types = args.types ?? INVESTIGABLE_TYPES;
```

Comment:
Should we have a security to be sure that args.types is included in Investigable types ?

Reply:
Good catch! I guess we do, I'll check what happen if we are not.